package.json

```diff
@@ -52,7 +52,6 @@
     "snyk:test:package": "yarn snyk:test --include",
     "start": "backstage-cli repo start",
     "start-backend": "echo \"Use 'yarn start example-backend' instead\"",
```

Comment:
I wonder how much value we're actually getting out of this one still being named example-backend instead of backend 🤔

Reply:
yep had the same thought